src/components/MenuItem/MenuItem.tsx

```diff
@@ -8,12 +8,27 @@ import theme from "@root/theme";
 import { BodyText } from "../Typography";
 
 const StyledMenuItem = styled(MUIMenuItem)`
-	min-width: 150px;
 
 	&.MuiMenuItem-root {
 		min-height: 42px;
 	}
 
+	&.css-kk1bwy-MuiButtonBase-root-MuiMenuItem-root.Mui-selected {
+		background-color: ${theme.colors.white};

```

Comment:
I don't think we really need to specify the background-color for just when it's selected. We could take this to the style from line 12 and that way it will always stay white.

Reply:
Ok, I have to use !important because by default MUI adds a background on the selected element